app.js

```diff
@@ -1,76 +1,40 @@
 require('dotenv').config();
+const fs = require('fs');
+const path = require('path');
 const Discord = require('discord.js');
-const client = new Discord.Client();
-
-client.once('ready', () => {
-    console.log('Ready');
-});
-
-client.login(process.env.BOT_TOKEN);
 
 // Use exclamation mark as the default prefix
 const prefix = process.env.PREFIX || '!';
-
-const help = '**Bot commands**\n✨ Start pairings with `!start`\n🛑 Stop pairings with `!stop`\n💖 Set your pronouns with `!pronouns PRONOUNS`\n💻 Share your GitHub profile with `!github GITHUB_URL`\n💼 Share your LinkedIn profile with `!linkedin LINKEDIN_URL`\n🐙 Share a fun fact about yourself with `!funfact FUN_FACT`\n❔ Show this menu with `!help`';
+const client = new Discord.Client();
 
 client.on('message', message => {
+    // Ignore messages that aren't for this bot to bother
+    if (!message.content.startsWith(prefix)) return;
     // Ignore other bot messages
     if (message.author.bot) return;
-
-    // Remove irc username suffix
-    const messageContent = message.content.replace(/<.*> /, '');
-
     // Ignore messages in public places (for now)
     if (message.channel.type !== 'dm') return;
 
```

Comment:
I think this block could be factored into a `isMessageIsInvalid` function

Reply:
Yeah sure. I am planning to make that into a separate PR because this one focuses on separating the commands into it's own file.